### ISSUE TITLE
Create sublist from selected - does not create sublist

### ISSUE THREAD

**Issue by janetc** [PTFS-Europe/rebus-list-2#332]
Testing first on Preview2 and then on Bucks.
1) Select materials and then click on the Create sublist from selected - 
2) then Move selected material  -
3) I enter the new sublist name and click Finish.
4) On the results page the name of the sublist displays with a green tick. Click Finish.
5) The page returns to Draft mode. The sublist isn't created and the materials have gone. 

If instead I choose Copy selected material (step 2) the sublist still isn't created (though the materials have not disappeared).

**Comment by AndrewIsh** [comment]
I cannot replicate this at all. I just did exactly the same steps as you on preview2 and it worked perfectly. Any chance you could try again?

**Comment by janetc** [comment]
I've done a couple of bugreplays for this - it may just be a workflow issue - if I click Finish on Step 4 the sublist is not created and the items disappear - see https://app.bugreplay.com/c/ptfseurope/report/1f3b75be-1cfb-4574-8dee-1af1ba0f5765.
If I click Go to List on Step 4 the sublist is created - see https://app.bugreplay.com/c/ptfseurope/report/1dfb8802-b4ef-4de3-8692-83c8a77bacfe

**Comment by AndrewIsh** [comment]
Right, I _think_ I know what you're getting at. So, in the first BugReplay, the sublist is in fact created in the background, it just doesn't look to be being appended to the list of sublists at the bottom of the list. When, in the second BugReplay, you click "Go to List", you are taken to the new sublist, this doesn't happen when you click "Finish", you just remain on the list you're currently working on.

The items disappear because you chose to "Move" them to the new sublist rather than copy. They're not in fact disappearing, they're just being moved into the new sublist.

So, I think the crux of the problem is that the new sublist isn't being appended to the list of sublists upon creation. I'll investigate.

**Comment by AndrewIsh** [comment]
https://github.com/PTFS-Europe/rebus-list/issues/2426

**Comment by hloliver** [comment]
Hi - I've just tried creating a sublist from selected items in a list and experienced the same issue (ie the new sublist wasn't appended to the main list, but the items disappeared from the main list because I'd opted to Move rather than Copy them.
Any idea when this will be fixed? We can work around it, but this is a nice easy way to create sublists from a large reading list!

**Comment by AndrewIsh** [comment]
Hi Helen

Thanks for this. I'm not sure at this stage when the fix will be in place. Until then, refreshing the page after the move should cause the new sublist to be displayed. Can you please confirm this is the case?

Many thanks
Andrew

**Comment by hloliver** [comment]
Hi Andrew

Afraid that didn't work, I tried refreshing/publishing/clearing browser history but no success

This is the list: https://tavi.rebuslist.co.uk/#/list/3756

and this is what is should have looked like: https://tavi.rebuslist.co.uk/#/list/3760 - I had to create these sublists first, then move the items into it.  But it was a bit more cumbersome than it sounds  as we've got a lot of lists that have 'Term 1' in the title, so had to rename the lists to make sure that I could find the right ones quickly, then rename them back again.

Helen

**Comment by AndrewIsh** [comment]
Hi Helen

I've just been doing some testing and it seems that if you click "Go to list" following the move then the sublist should be appended to correctly. Can you please try that and let me know.

Many thanks
Andrew

**Comment by hloliver** [comment]
Yes, that worked but you have to remember to expand the sublist in the Results pop-up otherwise you don't see the 'Go to list' option - I missed it the first time I tried and just clicked on Finish so lost the sublist and references again!
